front/app/containers/Admin/projects/project/analysis/InputPreview/index.tsx

```diff
@@ -75,10 +77,21 @@ const InputListItem = () => {
       ? additionalCustomFieldIds.filter((id) => id !== customFieldId)
       : [...(additionalCustomFieldIds || []), customFieldId];
 
-    updateAnalysis({
-      id: analysisId,
-      additional_custom_field_ids: newAdditionalCustomFieldIds,
-    });
+    updateAnalysis(
+      {
+        id: analysisId,
+        additional_custom_field_ids: newAdditionalCustomFieldIds,
+      },
+      {
+        onSuccess: () => {
```

Comment:
I don't think it matters, but just so you know, in a case like this (API call made), we could also have tracked it through the backend.